src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ConstructedEETypeNode.cs

```diff
@@ -137,11 +137,11 @@ protected override DependencyList ComputeNonRelocationBasedDependencies(NodeFact
 
             if (closestDefType.HasGenericDictionarySlot())
```

Comment:
I know this is pre-existing code, but should this just be `closestDefType.HasInstantiation`? `HasGenericDictionarySlot()` is a really confusing question to ask.